tests/phpunit/tests/oembed/WpEmbed.php

```diff
@@ -377,4 +379,16 @@ public function test_maybe_make_link_do_not_link_if_unknown() {
 		$this->wp_embed->linkifunknown = false;
 		$this->assertSame( $url, $this->wp_embed->maybe_make_link( $url ) );
 	}
+
+	/**
+	 * Test that oembed iframes are lazy loaded by default.
+	 *
+	 * @ticket 58773
```

Comment:
```suggestion
	 * Tests that oEmbed iframes are lazy loaded by default.
	 *
	 * @ticket 58773
	 *
	 * @covers WP_Embed::shortcode
```
- Using third-person singular verbs to begin docblock descriptions.
- Adding `@covers` annotation.